example-raft-kv/src/store/mod.rs

```diff
@@ -28,16 +28,29 @@ use serde::Deserialize;
 use serde::Serialize;
 use tokio::sync::RwLock;
 
-/// Request is a command to modify the state machine.
+/**
+ * Here you will set the types of request that will interact with the raft nodes. 
+ * For example the `Set` will be used to write data (key and value) to the raft database. 
+ * The `AddNode` will append a new node to the current existing shared list of nodes.
+ * You will want to add any request that can write data in all nodes here.
+ */
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum ExampleRequest {
     Set { key: String, value: String },
     AddNode { id: NodeId, addr: String },
 }
 
+// Inform to raft that `ExampleRequest` is an application data to be used by raft. 
 impl AppData for ExampleRequest {}
 
-/// The state after applied a request.
+/**
+ * Here you will defined what type of answer you expect from reading the data of a node.
+ * In this example it will return a optional value from a given key in 
+ * the `ExampleRequest.Set`.
+ * 
+ * TODO: SHould we explain how to create multiple `AppDataResponse`?
```

Comment:
Good point!

Maybe making it an `enum` would be better?
In a real-world application, there should be a corresponding enum variant in `ExampleResponse` for every enum variant in `ExampleRequest`.

Reply:
I think it's a good idea, we can make them similar to the Request enum. But I think we could leave that for another PR.